samples/rqd/blender/blender2.79-docker/Dockerfile

```diff
@@ -15,11 +15,11 @@ RUN yum -y install \
 
 # Download and install Blender 2.79
```

Comment:
I noticed the file path here also contains the version number. I think we can ditch the version-specific subdirectory so the file would live at `samples/rqd/blender/Dockerfile`.

Reply:
Resolved in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1259/commits/b9191eeb649d7c58369943664f1f144ba3619726